tools/admin/requests/permit-holder-information.ts

```diff
@@ -116,6 +144,20 @@ export type UpdatePermitHolderInformationResponse = {
   updateApplicationGeneralInformation: UpdateApplicationGeneralInformationResult;
 };
 
+/** Update permit holder information of application */
+export const UPDATE_NEW_PERMIT_HOLDER_INFORMATION = gql`
```

Comment:
```suggestion
export const UPDATE_NEW_APPLICATION_PERMIT_HOLDER_INFORMATION = gql`
```
Makes it clear that this is for new applications, not for creating new permit holders. Also please update the corresponding request/result types below